requirements.txt

```diff
@@ -1,9 +1,8 @@
-OWSLib>=0.11.0
-flake8>=3.0.4
-geopandas>=0.2.1
-jinja2>=2.7
-lxml>=3.5.0
-numpy>=1.11.2
-pandas>=0.19.0
-pyoos>=0.8.3
-shapely>=1.5.16
+OWSLib
+geopandas
+jinja2
+lxml
+numpy
+pandas
+pyoos
+shapely
```

Comment:
I removed all the pinning so Travis-CI can act like a testing background to catch failures early in case a new version breaks the code. (But there are no tests at the moment.)